# Information about the tracked person
std_msgs/Header header
bool person_detected
float32 confidence
float32 center_x
float32 center_y
float32 area
float32 distance_estimate  # Based on area
float32 horizontal_offset  # From frame center
